x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -124,13 +125,15 @@ When adding realms to a security domain, avoid authenticating with a newly-added
 
 Removing realms from a security domain can lead to unexpected behaviors
 and is not recommended.
-Resources created before the removal can be owned by different users depending on the resource type:
+Resources created or updated before the removal can be owned by different users depending on the resource type:
 
 - <<user-profile,User profiles>> are owned by the user for whom the profile was last
 <<security-api-activate-user-profile,activated>>.
 For users whose realms are no longer in the same domain as the owner user, a new user profile
 will be created for them next time the activate user profile API is called.
-- Resources such as API keys are owned by the user who originally created them.
+- An API key is owned by the user who originally <<security-api-create-api-key,created>> or last <<security-api-update-api-key,updated>> it.
+Users, including the original creator of the API key, will lose ownership, if their realms are no longer in the same domain as those of the current API key owner.
```

Comment:
```suggestion
Users, including the original creator of the API key, will lose ownership if their realms are no longer in the same domain as those of the current API key owner.
```

Reply:
I had a suspicion that comma didn't belong there. Cheers!